src/main/java/org/jenkinsci/plugins/workflow/cps/CpsFlowExecution.java

```diff
@@ -1441,7 +1441,11 @@ void notifyListeners(List<FlowNode> nodes, boolean synchronous) {
                 }
             } finally {
                 if (synchronous) {
-                    bc.abort(); // hack to skip save—we are holding a lock
+                    bc.abort(); //// hack to skip save—we are holding a lock
```

Comment:
Why the introduced `//`?

Reply:
Just to see if people are paying attention